result/rsvp.yaml

```diff
@@ -0,0 +1,192 @@
+openapi: 3.1.0
+
+info:
+  title: RSVP-TE metric results model.
+  version: ^0.0.0
+
+components:
+  schemas:
+    Rsvp.Metrics.Request:
+      description: >-
+        The request to retrieve RSVP-TE per Router metrics/statistics.
+      type: object
+      properties:
+        router_names:
+          description: >-
+            The names of RSVP-TE Routers to return results for.
+            An empty list as input will return results for all RSVP-TE routers.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Rsvp/properties/name"
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the input list is empty then all columns will be returned except for
+            any result_groups.            
+          type: array
+          items:
+            type: string
+            enum:
+              - ingress_p2p_lsps_configured
+              - ingress_p2p_lsps_up
+              - egress_p2p_lsps_up
+              - lsp_flap_count
+              - paths_tx
+              - paths_rx
+              - resvs_tx
+              - resvs_rx
+              - path_tears_tx
+              - path_tears_rx
+              - resv_tears_tx
+              - resv_tears_rx
+              - path_errors_tx
+              - path_errors_rx
+              - resv_errors_tx
+              - resv_errors_rx
+              - resv_conf_tx
+              - resv_conf_rx
+              - hellos_tx
+              - hellos_rx
+              - acks_tx
+              - acks_rx
+              - nacks_tx
+              - nacks_rx
+              - srefresh_tx
+              - srefresh_rx
+              - bundle_tx
+              - bundle_rx
+              - path_reevaluation_request_tx
```

Comment:
Is there no corresponding 'path_reevaluation_request_rx' stat?

Reply:
This is because test tools dont perform SPF which is needed to take action on a Path Re-evaluation request received by test tool. Hence, test tools are not expected to need to support handling of received Path Re-eval requests.